### PR TITLE
docs: publish v2.4 feature docs

### PR DESCRIPTION
**Pull request by stijnmoreels**
Publishes v2.4 feature docs.

### CODE REVIEW

File: docs/versioned_docs/version-v2.4/02-Features/correlation.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+```shell
+PM > Install-Package Arcus.Observability.Correlation
+```
+
+## What We Provide
+
+The `Arcus.Observability.Correlation` library provides a way to get access to correlation information across your application.
+What it **DOES NOT** provide is how this correlation information is initially set.
+
+It uses the the Microsoft dependency injection mechanism to register an `ICorrelationInfoAccessor` and `ICorrelationInfoAccessor<>` implementation that is available.
```
Comment (fgheysels) [inline docs/versioned_docs/version-v2.4/02-Features/correlation.md:27]:
```suggestion
It uses the Microsoft dependency injection mechanism to register an `ICorrelationInfoAccessor` and `ICorrelationInfoAccessor<>` implementation that is available.
```

File: docs/versioned_docs/version-v2.4/03-Guidance/use-with-dotnet-and-functions.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+---
+title: "Using Arcus & Serilog in .NET Core and/or Azure Functions"
+layout: default
+---
+
+# Using Arcus & Serilog in .NET Core and/or Azure Functions
```
Comment (fgheysels) [inline docs/versioned_docs/version-v2.4/03-Guidance/use-with-dotnet-and-functions.md:6]: Is it still relevant to say 'in .NET Core'  , now that we also support .NET 6 ?

File: docs/versioned_docs/version-v2.4/01-index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+layout: default
+slug: /
+sidebar_label: Welcome
+---
+
+[![NuGet Badge](https://buildstats.info/nuget/Arcus.Observability.Correlation?includePreReleases=true)](https://www.nuget.org/packages/Arcus.Observability.Correlation/)
+
+# Installation
+
+The Arcus.Observability.Correlation can be installed via NuGet:
```
[the remark this answers is not in the document]
Comment (stijnmoreels) [reply docs/versioned_docs/version-v2.4/01-index.md:12]: Hmm, we should probably add this to all versions, then.

File: docs/versioned_docs/version-v2.4/02-Features/correlation.md (hunk 1 above)
Comment (stijnmoreels) [reply docs/versioned_docs/version-v2.4/02-Features/correlation.md:27]: Again, should be on all, I'll add an issue for this.

File: docs/versioned_docs/version-v2.4/03-Guidance/use-with-dotnet-and-functions.md (hunk 2 above)
Comment (stijnmoreels) [reply docs/versioned_docs/version-v2.4/03-Guidance/use-with-dotnet-and-functions.md:6]: Yeah, that's indeed something that's different now. Hmm, if we change the title, we should make sure that we add an redirect bc it's a H1 title.

Comment (stijnmoreels) [reply docs/versioned_docs/version-v2.4/03-Guidance/use-with-dotnet-and-functions.md:6]: `Using Arcus & Serilog in Azure Functions`?

Comment (stijnmoreels) [reply docs/versioned_docs/version-v2.4/03-Guidance/use-with-dotnet-and-functions.md:6]: Updated this.

**Review by fgheysels — APPROVED** [review]
Fine for me!

**Comment by stijnmoreels** [comment]
> Fine for me!

Great! Updated the links on the index page, too.

**Pull request merged**